src/EnvConfig.php

```diff
@@ -28,6 +29,8 @@ class EnvConfig implements SiteConfig
         'preprod' => self::STAGING,
         'pre-prod' => self::STAGING,
         'pre-production' => self::STAGING,
+        'training' => self::TRAINING,
+        'train' => self::TRAINING,
```

Comment:
This is something very specific to one of our clients (this repo is public so I'm mentioning it, but you know). I don't think it should be in this generic and public package.

Besides that, the value of the array is supposed to be a value returned by `wp_get_environment_type()` so we can map arbitrary env names to one of the 4 supported by WP, and "training" is _not_ supported by VIP.